### DISCUSSION TITLE
Listbox inside modals lose focus

### DISCUSSION THREAD

**Discussion by emiCacc** [primefaces/community#1511] [PrimeNG]
Hello everyone, I'm having trouble manipulating the tabindex within my modals when passing through a Listbox.

There are several scenarios that I have, first of all this is a nested modal, where I have a app-componentParent and inside it a app-modal1 and inside this a app-modal2 and it is in this (modal 2 ) where I have the problem.

Neither using the focusTrap component nor setting the tabIndex manually can I achieve the normal TAB flow.

I've had to capture the elements that I want to be 'tabable' and bring them in with ViewChilds and listeners so I can schedule the flow.

I do not know what else to do.

This problem arose after updating to version 17.2 of Angular, although I'm not sure if that was the cause.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by cetincakiroglu** [comment]
Closing due to the inactivity.
